substrate/client/network/src/litep2p/discovery.rs

```diff
@@ -446,8 +470,10 @@ impl Discovery {
 	}
 
 	/// Start providing `key`.
-	pub async fn start_providing(&mut self, key: KademliaKey) {
-		self.kademlia_handle.start_providing(key.into()).await;
+	pub async fn start_providing(&mut self, key: KademliaKey) -> QueryId {
```

Comment:
We add here the return value. 
Are there other places in the codebase which could be impacted with this?